src/pages/configurePatch/configurePatchCore/ConfigureBuildVariants.tsx

```diff
@@ -106,11 +154,13 @@ export const cardSidePadding = css`
 const Container = styled.div`
   ${cardSidePadding}
 `;
-const StyledSiderCard = styled(SiderCard)`
+const StyledSiderCard = styled(SiderCard)<StyledSiderCardProps>`
```

Comment:
The issue you mentioned about the error with the props is due to how leafygreen uses emotion that messes with how props are propagated. They provide a custom version of emotion to fix this. 
https://github.com/mongodb/leafygreen-ui/tree/main/packages/emotion

An alternative is to just move the props to another non leafygreen element

Reply:
went with the alternative